### DISCUSSION TITLE
[Feature] smart search for videos enhancements

### DISCUSSION THREAD

**Discussion by Matszwe02** [immich-app/immich#10789] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It will be helpful to make smart search work for videos.

My suggestion is to take a few photos of a video, and run CLIP for each of them. Then, there could be voice recognition (preferably something like whisper) to detect what people say on a given video, which is very helpful for searching for a particular one.

I hope that's doable and not very hard to implement, yet it could be a very powerful addition 

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#5936
